spec/transfers_spec.rb

```diff
@@ -66,7 +66,7 @@
     request_body = JSON.parse(json_from_file("mocks/requests/Transfers/make_transfer.json"))
     response_body = json_from_file("mocks/responses/Transfers/create_transfer_request_requires_sca.json")
 
-    url = @shared_values[:client].service_url("Transfers", "transfers", "3")
+    url = @shared_values[:client].service_url("Transfers", "transfers", "4")
```

Comment:
Oke, toch 1 wijziging moeten doen. Dit is een spec die we zelf hebben toegevoegd. De default versie is nu 4. Dus daar wordt tegenaangetest. En de stub ging nog naar v3, waardoor de spec faalde.